specialists/agents/anxiety/data_manager.py

```
@@ -0,0 +1,962 @@
+        )
+    
+    async def _save_agent_with_new_goal(self, agent: Any):
+        """Save agent back to DB when a new goal is added"""
+        try:
+            self.mongo_client.db.anxiety_agents.update_one(
+                {"user_profile_id": agent.user_profile_id},
+                {"$set": agent.model_dump()},
+                upsert=True
+            )
+            logger.debug(f"Saved agent with new goal for user {agent.user_profile_id}")
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)